Gemfile

```diff
@@ -66,6 +66,7 @@ gem 'rack-attack', '~> 4.2.0'
 gem 'ahoy_matey'
 gem 'oj'
 gem 'oj_mimic_json'
+gem 'custom_counter_cache', github: 'gdpelican/custom_counter_cache'
```

Comment:
I was running into a bug in custom_counter_cache, see here: https://github.com/cedric/custom_counter_cache/pull/11